pkg/api/operators/apis/experiment/v1alpha2/util.go

```diff
@@ -73,10 +73,34 @@ func (exp *Experiment) IsFailed() bool {
 	return hasCondition(exp, ExperimentFailed)
 }
 
+func (exp *Experiment) IsRunning() bool {
+	return hasCondition(exp, ExperimentRunning)
+}
+
+func (exp *Experiment) IsRestarting() bool {
+	return hasCondition(exp, ExperimentRestarting)
+}
+
 func (exp *Experiment) IsCompleted() bool {
 	return exp.IsSucceeded() || exp.IsFailed()
 }
 
+func (exp *Experiment) GetCurrentConditionType() ExperimentConditionType {
+	if exp.IsCreated() {
```

Comment:
What would this function used for?  If you want the latest condition, it would be better to directly lookup the Type of the last status.conditions?

Reply:
Is that true that the latest `Condition` in this array: https://github.com/kubeflow/katib/blob/master/pkg/api/operators/apis/experiment/v1alpha2/experiment_types.go#L77, will be current? 
I need to get Current Condition of Experiment.